omero/sysadmins/publication.rst

```diff
@@ -0,0 +1,96 @@
+Complete workflow for hosting data for a publication
+====================================================
+
+The following describes the steps of a complete workflow for using OMERO to
+host public data associated with a publication. It is illustrated using an
+example publication from our lab in Dundee,
+`Schleicher et al, 2017 <http://dx.doi.org/10.1098/rsob.170099>`_.
+
+Group setup
+-----------
+
+#. Create a dedicated :doc:`public group <public>` to host the raw data
+   underlying the publication.
+#. Add all the authors of the paper to this new public group.
+
+Data setup
+----------
+
+#. Move the original images into the dedicated group using OMERO.web or
+   :doc:`OMERO.cli </users/cli/chgrp>`. The CLI is best used where Images or
+   Datasets are cross-linked to other Datasets or Projects in the original
+   group. The command ``bin/omero chgrp Project:$ID --include Dataset,Image``
+   cuts the cross-links in the original group and preserves the
+   Project/Dataset/Image hierarchy prepared for the move by the author.
+#. If you have used OMERO.figure to create your figures for publication, you
+   can always find the original data by using the 'info' tab, as shown in the
+   :help:`OMERO.figure Help guide <figure.html#info>` (OMERO.figure supports a
+   complete figure creation workflow, including exporting figures into image
+   processing applications for final adjustments - see the
+   :help:`OMERO.figure Help guide <figure.html>` for full details).
+#. Having all the data belong to one user simplifies the UI experience for
+   public users. If necessary, ownership of data can be transferred using the
+   'Chown' privilege (see :doc:`restricted-admins` and
+   :doc:`/users/cli/chown`).
+
+Configuring OMERO.web
+---------------------
+
+#. Configure the OMERO.web filter on the public user by setting the public URL
+   filter to allow 'webclient' (see :doc:`public` and
+   :property:`omero.web.public.url_filter`) so that the full webclient is
+   visible for the public user, and thus also the Data tree with Projects and
+   Datasets is browsable as well as Tags tab and full viewer of the images
+#. In order to have both a public webclient and a closed webclient side by
+   side without affecting the existing closed webclient (and thus giving your
+   existing non-public OMERO users an unchanged user experience in their
+   workflows), a dedicated,
+   :doc:`separate web server <unix/install-web/web-deployment>` for servicing
+   the public workflows can be added and configured to point at an existing
```

Comment:
Maybe rather "the existing OMERO.server." Definitely we should assume the users already have an existing OMERO.server, and that is the one they should point the new webserver to.